packages/appstore/src/components/create-wallet/create-wallet-wrapper.tsx

```diff
@@ -0,0 +1,26 @@
+import { observer } from 'mobx-react-lite';
+import React from 'react';
+import { useStores } from 'Stores';
+import CreateWallet from './create-wallet';
+import './create-wallet.scss';
+
+type TProps = {
+    is_dark_mode_on: boolean;
+    should_show_fiat: boolean;
+};
+
+const CreateWalletWrapper = ({ is_dark_mode_on, should_show_fiat }: TProps) => {
+    const { wallet_store } = useStores();
+
+    React.useEffect(() => {
+        wallet_store.onMount();
+
+        return () => wallet_store.onUnmount();
```

Comment:
why are we clearing `wallet_names` onUnmount? 
We don't have to clear `wallet_names` onUnmount. Let's keep the `wallet_names` inmemory, it might be used in other components.